sdk/src/main/java/io/dapr/client/DaprClient.java

```diff
@@ -405,83 +409,83 @@
    * @param <T>     The Type of the return.
    * @return A Mono Plan for the requested State.
    */
-  <T> Mono<Response<List<State<T>>>> getStates(GetStatesRequest request, TypeRef<T> type);
+  <T> Mono<Response<List<State<T>>>> getBulkState(GetStatesRequest request, TypeRef<T> type);
```

Comment:
@artursouza should the domain class `GetStatesRequest` also be renamed to `GetBulkStateRequest` ?

Reply:
Please, rename request classes too.